src/Tribe/Models/Post_Types/Venue.php

```diff
@@ -41,7 +43,8 @@ protected function build_properties( $filter ) {
 				'state'                 => $state,
 				'province'              => $province,
 				'zip'                   => $zip,
-				'permalink'             => get_permalink( $this->post->ID ),
+				'permalink'             => $permalink,
+				'directions_link'       => esc_url( $directions_link ),
```

Comment:
Is this escaping too early? Should we do `esc_url_raw()` instead? Or nothing here and escape late.

Reply:
Agreed.

Reply:
I think the method is already escaping the return.